plugins/utilities.json

```diff
@@ -3,6 +3,101 @@
   "description": "Utilities",
   "plugins_base_url": "https://github.com/bombsquad-community/plugin-manager/{content_type}/{tag}/plugins/utilities",
   "plugins": {
+    "sandbox": {
+      "description": "The most advanced bombsquad mod ever imo, which can basically do almost everything in the game, literally.",
```

Comment:
will it let me pet cats in-game? if not, let's try be a lil more specific plwiss. :3

Reply:
let's not do opinions in descriptions, nicer would be to phrase descriptions in a way as if they're supposed to tell us facts on what the plugin specifically does.

Reply:
how about going outside and petting a real one?
conversation to outdated ultilities.json resolved.